### PR TITLE
add mount point to quota warning message

### PR DESCRIPTION
**Pull request by icewind1991**
makes it more clear to the user what the quota applies to

See also https://github.com/nextcloud/server/pull/22421

### CODE REVIEW

**Review by rullzer — APPROVED** [review]

**Review by skjnldsv — APPROVED** [review]

**Comment by skjnldsv** [comment]
Broken tests on master :warning:

**Comment by MorrisJobke** [comment]
> Broken tests on master ⚠️

Fixed in #23193

File: lib/private/legacy/OC_Helper.php [hunk 1]
```diff
@@ -545,6 +545,7 @@ public static function getStorageInfo($path, $rootInfo = null) {
 		if ($owner) {
 			$ownerDisplayName = $owner->getDisplayName();
 		}
+		[,,,$mountPoint] = explode('/', $mount->getMountPoint(), 4);
```
Comment (nickvergessen) [inline lib/private/legacy/OC_Helper.php:548]: Get the following error printed when running integration tests:
```
{"Exception":"Error","Message":"Undefined offset: 3 at \/home\/nickv\/Nextcloud\/21\/server\/lib\/private\/legacy\/OC_Helper.php#548","Code":0,"Trace":[{"file":"\/home\/nickv\/Nextcloud\/21\/server\/lib\/private\/legacy\/OC_Helper.php","line":548,"function":"onAll","class":"OC\\Log\\ErrorHandler","type":"::"},{"file":"\/home\/nickv\/Nextcloud\/21\/server\/apps\/provisioning_api\/lib\/Controller\/AUserData.php","line":190,"function":"getStorageInfo","class":"OC_Helper","type":"::"},{"file":"\/home\/nickv\/Nextcloud\/21\/server\/apps\/provisioning_api\/lib\/Controller\/AUserData.php","line":137,"function":"fillStorageInfo","class":"OCA\\Provisioning_API\\Controller\\AUserData","type":"->"},{"file":"\/home\/nickv\/Nextcloud\/21\/server\/apps\/provisioning_api\/lib\/Controller\/UsersController.php","line":394,"function":"getUserData","class":"OCA\\Provisioning_API\\Controller\\AUserData","type":"->"},{"file":"\/home\/nickv\/Nextcloud\/21\/server\/lib\/private\/AppFramework\/Http\/Dispatcher.php","line":218,"function":"getUser","class":"OCA\\Provisioning_API\\Controller\\UsersController","type":"->"},{"file":"\/home\/nickv\/Nextcloud\/21\/server\/lib\/private\/AppFramework\/Http\/Dispatcher.php","line":127,"function":"executeController","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"\/home\/nickv\/Nextcloud\/21\/server\/lib\/private\/AppFramework\/App.php","line":157,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"\/home\/nickv\/Nextcloud\/21\/server\/lib\/private\/Route\/Router.php","line":308,"function":"main","class":"OC\\AppFramework\\App","type":"::"},{"file":"\/home\/nickv\/Nextcloud\/21\/server\/ocs\/v1.php","line":88,"function":"match","class":"OC\\Route\\Router","type":"->"},{"file":"\/home\/nickv\/Nextcloud\/21\/server\/ocs\/v2.php","line":24,"args":["\/home\/nickv\/Nextcloud\/21\/server\/ocs\/v1.php"],"function":"require_once"}],"File":"\/home\/nickv\/Nextcloud\/21\/server\/lib\/private\/Log\/ErrorHandler.php","Line":98,"CustomMessage":"--"}
```

**Pull request merged**